src/Polly/Guard.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Runtime.CompilerServices;
+
+static class Guard
```

Comment:
I prefer to be explicit that have ambiguity of maybe someone forgot to add it.

```suggestion
internal static class Guard
```